tests/Backend/CommonPart1/BucketsTest.php

```diff
@@ -575,6 +576,50 @@ public function testBucketExists(string $devBranchType, string $userRole): void
         $this->assertFalse($this->_testClient->bucketExists('in.ukulele'));
     }
 
+    /**
+     * @dataProvider provideComponentsClientTypeBasedOnSuite
+     */
+    public function testBucketOwner(string $devBranchType, string $userRole): void
+    {
+        $this->initEvents($this->_testClient);
+
+        $bucketName = 'bucketOwnerTesting';
+
+        $this->dropBucketIfExists($this->_testClient, "in.c-{$bucketName}", true);
+        $bucketId = $this->_testClient->createBucket($bucketName, self::STAGE_IN);
+
+        $bucket = $this->_testClient->getBucket($bucketId);
```

Comment:
@vojtabiberle napadlo me jeste, ze by se tady hodily dva male asserty (udelal bych to klidne tady v jednom...)
1. getBucketDetail na prazdnem bucektu -> mel bys dostat BucketOwnerNotFoundException
2. updateBucketOwner s invalidnim adminem -> BucketOwnerNotFoundException
doplnil bys to pls jeste? sorry, doslo mi to az rano

Reply:
přidáno